services/wfm/src/main/java/org/openkilda/wfm/topology/utils/KibanaLogManager.java

```diff
@@ -0,0 +1,126 @@
+/* Copyright 2018 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.wfm.topology.utils;
+
+import org.openkilda.messaging.info.event.IslChangeType;
+import org.openkilda.messaging.info.event.SwitchState;
+import org.openkilda.messaging.model.SwitchId;
+
+import lombok.extern.slf4j.Slf4j;
+import org.slf4j.MDC;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * The KIBANA dashboard log manager.
+ */
+@Slf4j
+public class KibanaLogManager {
+
+    private static final String SWITCH = "switch";
```

Comment:
The constant name should indicate the purpose or scope or meaning of the constant. This one is unclear.